examples/example_plugin.rs

```diff
@@ -0,0 +1,93 @@
+use fender::{
+    declare_plugin, fender_value::FenderValue, fndr_native_func, plugin::Plugin,
+    type_sys::type_system::FenderTypeSystem,
+};
+use freight_vm::{expression::NativeFunction, function::ArgCount};
+use std::collections::HashMap;
+
+#[derive(Debug)]
+struct MyPlugin {
+    example_1: (NativeFunction<FenderTypeSystem>, ArgCount),
+    example_2: (NativeFunction<FenderTypeSystem>, ArgCount),
```

Comment:
Can this be turned into a named type?

Reply:
wdym?

Reply:
`(NativeFunction<FenderTypeSystem>, ArgCount)` should be a struct

Reply:
oh I thought you meant the `Plugin`